src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -883,6 +897,10 @@ IS 'filename of the requested media object, possibly migrated from moj_transform
 COMMENT ON COLUMN media_request.output_format
 IS 'format of the requested media object, possibly migrated from moj_transformation_s';
 
+CREATE TABLE media_type
+(met_id                      INTEGER                       NOT NULL
+,media_type                  CHARACTER VARYING             NOT NULL

```

Comment:
Wondering whether this would be any different from the mime type presumably in media table
`media_format                CHARACTER VARYING             NOT NULL`?

Reply:
I agree, media_format should be sufficient to determine what might possibly be held in media_type.  If absolutely necessary, we might normalise media_format, to have an attribute to include acceptable values of { audio/video/... }. Awaiting clarification.